### DISCUSSION TITLE
Contour on windows

### DISCUSSION THREAD

**Discussion by montyvesselinov** [contour-terminal/contour#638] [Development]
I ma very new here.

I am building contour both on windows 11 and WSL2 ubuntu.

I have still issues to build it on windows 11.

I was able to build on WSL2 ubuntu.

Howerver, the copy & paste does not seem to work between the UNIX and Windows. Also, I cannot make it fancy.  Are the profiles that i can use?

**Comment by christianparpart** [comment]
Hey, sorry for the late response, I am not sure how on earth I skipped that in my notifications box. (same for the other topic).

So does this mean you run Contour in WSL2 and use some X11 server on Windows native to render to? What's your keybinds for copy'n'paste? Maybe it is also an X server config thing that allows/forbids copy'n'paste?

What would you like for the fancyness. What are you missing? You can configure background image, have that one conditionally blurred, or transparent background and have that one conditionally blurred. That's all configurable.

Your chosen font can be one that supports ligatures (I use "Jet Brains Mono Nerd Font Mono" in fact, which is lovely, but that's pure taste).

With regards to fancyness, I suggest you trying out sixel images. That will work on UNIX, including WSL2 when X11-forwarded but sadly not on ConPTY (due to ConPTY - not because of us).
